tests/test_value_range.py

```diff
@@ -97,16 +98,16 @@ def test_float_1(self):
 
 class TestPickValueFromRandomizedRange(unittest.TestCase):
     def __init__(self, *args, **kwargs):
-        super(TestPickValueFromRandomizedRange, self).__init__(*args, **kwargs)
-        self.session = tf.Session()
-        self.clock_ph = tf.placeholder(dtype=tf.float64, name='clock')
+        super().__init__(*args, **kwargs)
+        self.session = tfv1.Session()
 
     def _ending_tester(self, value_range, clock_min, clock_max, expected_min, expected_max):
         with self.session as session:
-            tf_pick = tf_pick_value_from_range(value_range, clock=self.clock_ph)
+            clock_ph = tfv1.placeholder(dtype=tf.float64, name='clock')
```

Comment:
that looks weird, we create the placeholder at the end now, instead of in the `__init__`?